packages/atomic/cypress/fixtures/test-recs-fixture.ts

```diff
@@ -0,0 +1,262 @@
+import {
+  TestFeature,
+  SearchResponseModifier,
+  SearchResponseModifierPredicate,
+  interceptSearchResponse,
+  TestFixture,
+} from './test-fixture';
+import {i18n} from 'i18next';
+import {buildTestUrl} from '../utils/setupComponent';
+import {AnalyticsTracker, AnyEventRequest} from '../utils/analyticsUtils';
+
+export type RecsInterface = HTMLElement & {
+  initialize: (opts: {
+    accessToken: string;
+    organizationId: string;
+  }) => Promise<void>;
+  getRecommendations: () => void;
+  i18n: i18n;
+  language: string;
+};
+
+export class TestRecsFixture {
```

Comment:
It seems like quite a bit of copy paste from `TestFixture`... 

Could we refactor it a bit to share more code ? Should not be too complicated 🤔

Reply:
Yes I've exported a bit more as of now